masterlist.yaml

```
@@ -6410,3 +6410,44 @@ plugins:
         crc: 0x6E9E192A
         util: '[FO4Edit v4.0.4b](https://www.nexusmods.com/fallout4/mods/2737)'
         itm: 4
+
+  - name: '__Fortune_Finder_Leveling.esp'
+    clean:
+      - crc: 0x3336af55
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+    url:
+      - 'https://www.afkmods.com/index.php?/files/file/2410-fortune-finder-leveling/'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4241028'
```

For consistency, the `url` key should be sorted above the `clean` key, a trailing slash should be added to the Bethesda.net Mods URL, & the cleaning data checksum should be capitalized. Also, the locations would benefit from names as they can be used in hyperlinks when displaying them:

```suggestion
    url:
      - link: 'https://www.afkmods.com/index.php?/files/file/2410-fortune-finder-leveling/'
        name: 'Fortune Finder Leveling (AFK Mods)'
      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4241028/'
        name: 'Fortune Finder Leveling (Bethesda.net Mods)'
    clean:
      - crc: 0x3336AF55
        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
```